deployments/biology/image/environment.yml

```diff
@@ -5,13 +5,16 @@ dependencies:
 - python=3.8.*
 - pip=20.*
 
+# nodejs
+- nodejs
```

Comment:
For R, we should get nodejs from ubuntu - that's what some of the R packages that use js want (particularly, `rstan`). This gives us an older nodejs but that's fine.